tensorflow-serving-api/meta.yaml

```diff
@@ -21,12 +21,11 @@ requirements:
   host:
    - bazel {{ bazel }}
    - python # noarch package; don't tie to specific python version
-   - tensorflow-base {{ tensorflow }}
+   - numpy {{ numpy }}
    - grpcio {{ grpcio }}
    - protobuf {{ protobuf }}
   run:
    - python # noarch package; don't tie to specific python version
-   - tensorflow-base >={{ tensorflow }}
```

Comment:
Maybe this one is still needed. You can install `tensorflow-serving-api` without `tensorflow-serving`, but you do need `tensorflow` for `tensorflow-serving-api` makes sense.

Reply:
Yeah, that's why I'd kept runtime dependency initially. But later on tried removing it from run section too and it worked.
But yes, I agree although I could launch `tensorflow_model_server` without tensorflow being installed, it won't make any sense without TF. However, I think TF can be installed separately by user in the environment . We don't have to force it by adding runtime dependency.

Reply:
ok, that's fine.